Framework/Geometry/src/Instrument/DetectorGroup.cpp

```diff
@@ -454,5 +455,15 @@ size_t DetectorGroup::index() const {
   throw std::runtime_error("A DetectorGroup cannot have an index");
 }
 
+void DetectorGroup::registerContents(
```

Comment:
As far as I know a `DetectorGroup` is never part of an instrument, should we simply throw here?